include/seqan3/utility/bloom_filter/bloom_filter.hpp

```diff
@@ -0,0 +1,385 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Tobias Loka <tobias.loka AT hpi.de>
+ * \brief Provides seqan3::bloom_filter.
+ */
+
+#pragma once
+
+#include <seqan3/std/algorithm>
+#include <seqan3/std/bit>
+#include <seqan3/search/dream_index/interleaved_bloom_filter.hpp>
+
+#include <sdsl/bit_vectors.hpp>
+
+#include <seqan3/core/concept/cereal.hpp>
+#include <seqan3/core/detail/strong_type.hpp>
+
+namespace seqan3
+{
+
+/*!\brief The bloom filter. A data structure that efficiently answers set-membership queries.
+ * \tparam data_layout_mode_ Indicates whether the underlying data type is compressed. See seqan3::data_layout.
+ * \implements seqan3::cerealisable
+ *
+ * \details
+ *
+ * ### Bloom Filter (BF)
+ *
+ * The [Bloom Filter](https://en.wikipedia.org/wiki/Bloom_filter) is a probabilistic data structure.
+ * A Bloom Filter can be thought of as a bitvector of length `n` and `h` hash functions and is used to determine set
+ * membership. To insert data, the data is hashed by the `h` hash functions (returning values in `[0, n)`) and the
+ * corresponding `h` positions in the bitvector are set to `1`. To query data, i.e. to determine whether the query
+ * belongs to the set the Bloom Filter was built for, the query is hashed by the same `h` hash functions and the
+ * corresponding positions are checked. If all `h` positions contain a `1`, the query is (probably) in the data set.
+ * Since the Bloom Filter has variable length, the hashing is not bijective, i.e. it may return true for a set
+ * membership query even though the query was never inserted into the Bloom Filter. Note that the Bloom Filter
+ * will always return `true` if the query was inserted, i.e. there may be false positives, but no false negatives.
+ * 
+ * ### Querying
+ * To query the Bloom Filter for a value, call seqan3::bloom_filter::contains(value) which returns
```

Comment:
```suggestion
 * To query the Bloom Filter for a value, call seqan3::bloom_filter::contains which returns
```

It doesn't link, and I think you do not name the parameter.
You can check if it works by building the documentation or visiting the preview build posted by the vercel bot (https://seqan3-git-fork-tloka-bloomfilter-seqan.vercel.app/usr/html/classseqan3_1_1bloom__filter.html in this case)